File: AbaxXBRLWeb/abax-xbrl/shared/modelos/OptionItemDto.ts
module abaxXBRL.shared.modelos {

    /**
    * Elemento para un arreglo de opciones de un capo de selección.
    **/
    export class OptionItemDto implements Serializable<OptionItemDto> {

        /**
        * Valor de la opción
        */
        public Value: string;

        /**
        * Etiqueta de la opción.
        **/
        public Label: string;

        /**
        * Deserializa un elemennto de entrada aun objeto OptionItemDto.
        * @param input Elemento con la información a deserializar.
        * @return Objeto de tipo OptionItemDto con la información deserializada.
        **/
        public deserialize(input: any): OptionItemDto {

            this.Value = input.Value;
            this.Label = input.Label;

            return this;
        }

    }

}

File: AbaxXBRLWeb/Common/Aplication/Dto/OptionItemDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace AbaxXBRLViewerWeb.Common.Aplication.Dto
{
    /// <summary>
    /// Elemento para un arreglo de opciones de un capo de selección.
    /// </summary>
    public class OptionItemDto
    {
        /// <summary>
        /// Valor de la opción
        /// </summary>
        public string Value { get; set; }
        /// <summary>
        /// Etiqueta de la opción.
        /// </summary>
        public string Label { get; set; } 
    }
}